docs/collator-instructions.md

```diff
@@ -0,0 +1,177 @@
+
+``sudo systemctl stop tinkernet-collator.service``
+
+``ls /var/lib/tinkernet``
+
+You should see "chains" and "polkadot" directories.
+
+``sudo apt install curl lz4 tar``
+
+Enter y to continue if prompted.
+
+``sudo rm -rf /var/lib/tinkernet/polkadot/chains/ksmcc3/*``
+
+``sudo curl -o - -L https://ksm-rocksdb.polkashots.io/snapshot | sudo lz4 -c -d - | sudo tar -x -C /var/lib/tinkernet/polkadot/chains/ksmcc3``
+
+Once that's downloaded, we need to make sure you add your account to the collator, I would strongly reccomend making a new account for that... go to polkadot.js.org, and make a new account but save the "raw seed", and not the mnemonic.
+
+``sudo /usr/local/bin/tinkernet-collator key insert --base-path /var/lib/tinkernet --chain /var/lib/tinkernet/tinker-raw.json --scheme Sr25519 --suri "your_private_key(RAW SEED)_here" --password-interactive --key-type aura``
+
+Note: see more here if you want to double check the above https://docs.substrate.io/tutorials/get-started/trusted-network/#add-keys-to-the-keystore
+
+Now we can start the the service again:
+
+``sudo systemctl start tinkernet-collator.service``
+
+Now we need to rotate keys and set our keys on chain.
+
+Ensure the collator is running, or this step won't work:
+
+``curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' http://127.0.0.1:9933/``
+
+You will be greeted with an output that looks like:
+
+``{"jsonrpc":"2.0","result":"0xprivate_key_will_be_here0","id":1}``
+
+"result":"**0x_private_key_will_be_here0**" is what we are interested in.
+
+You need to make sure that you have a Polkadot/Substrate account set up, here's some videos in case you don't know how to do that:
+
+1. Polkadot JS Video  https://www.youtube.com/watch?v=dG0DP9vayPY    https://www.youtube.com/watch?v=BpTQBAyFvEk
+
+2. Talisman Video   https://docs.talisman.xyz/talisman/talisman-initiation/setup-a-talisman-wallet  
+
+Now that you have made an account using one of those extensions, head on over to the GM Parachain section of Polkadot JS: 
+
+https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.gmordie.com#/extrinsics
+
+Ensure that you are in the Developer pallet (the top header), and navigate to the extrinsics section in the drop down.
+
+Note: If you are doing this before the token distribution event, please ping the GM Intern in our Discord, and I will send you some $FREN so you can send the extrinsics.
+
+In the "using the selected account field" select the account you just made for the collator.<br/>
+In the "submit the following extrinsic field" select "session".<br/>
+In the next field (to the right), select "setKeys(keys, proof)".<br/>
+In the "keys:" field, paste in your **0x_private_key_will_be_here0** from your node.<br/>
+In the "proof" field, type in " 0 ".
+
+Submit the transaction.
+
+Congratulations, you should now be onboarded as a collator.
```

Comment:
Missing steps to register as a candidate.

Reply:
✅ done